shared/validate/schema/model/transaction/create/vote.ts

```diff
@@ -47,7 +43,13 @@ export const COMPONENTS_TRS_VOTE = [
                     },
                     secret: {
                         type: 'string',
-                        minLength: 1
+                        minLength: 20,
```

Comment:
It is incorrect, secret length is near 100 symbols

Reply:
need create special validator for secret. It`s 13 words.